pom.xml

```diff
@@ -977,6 +978,12 @@
                 <version>${flink.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>io.streamnative.connectors</groupId>
+                <artifactId>pulsar-flink-connector_2.11</artifactId>
```

Comment:
The scala version of 2.11 can be replaced by variables?

Reply:
done